streams/src/test/java/org/apache/kafka/streams/kstream/KStreamBuilderTest.java

```diff
@@ -50,6 +51,7 @@
 import static org.junit.Assert.assertTrue;
 
 @SuppressWarnings("deprecation")
+@Ignore
```

Comment:
Why this?

Reply:
Since `KStreamsBuilder` is deprecated and going away at some point, I did not include it in the graph building process, hence the tests are ignored as all calls for building the topology go through the graph building process. 

Maybe this needs a deeper discussion offline.